src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+type Filter = 'all' | 'active' | 'completed';
+
+interface FooterProps {
+  activeCount: number;
+  completedCount: number;
+  filter: Filter;
+  setFilter: (filter: Filter) => void;
+  onClearCompleted: () => void;
+}
+
+export const Footer: React.FC<FooterProps> = ({
+  activeCount,
+  completedCount,
+  filter,
+  setFilter,
+  onClearCompleted,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeCount} items left
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={`filter__link${filter === 'all' ? ' selected' : ''}`}
+          data-cy="FilterLinkAll"
+          onClick={e => {
+            e.preventDefault();
+            setFilter('all');
+          }}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={`filter__link${filter === 'active' ? ' selected' : ''}`}
+          data-cy="FilterLinkActive"
+          onClick={e => {
+            e.preventDefault();
+            setFilter('active');
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method